pontoon/machinery/views.py

```diff
@@ -14,6 +14,7 @@
 
 from pontoon.base import utils
 from pontoon.base.models import Locale, TranslationMemoryEntry
+from django.http import HttpResponse
```

Comment:
We already import from `django.http` a few lines up, so the line 10 should read like:
`from django.http import HttpResponse, HttpResponseBadRequest, JsonResponse`